spec/requests/shopping_list_items_spec.rb

```diff
@@ -123,8 +123,8 @@
               # has frozen because Rails (Postgres?) sets the last three digits of
               # the timestamp to 0, which was breaking stuff in CI (but somehow not
               # in dev).
-              expect(shopping_list.reload.updated_at).to be_within(0.25.seconds).of(t)
-              expect(master_list.reload.updated_at).not_to be_within(0.25.seconds).of(t)
+              expect(shopping_list.reload.updated_at).to be_within(0.05.seconds).of(t)
```

Comment:
Narrowed down the tolerances here because we know the rounding error in CI is considerably less than 0.05s and we want to get as close as possible.